### PR TITLE
Feat/menu

### LINKED ISSUE #3: ## Set up a Menu
As a user,
I want to navigate through the app,
so I can use all features

- [ ] implement a grid with cards 
- [ ] set up Cards for every menu point 
- [ ] link the Cards to their particular destination

### PR DESCRIPTION
**Pull request by Marc-Klein**
## Motivation

- add a basic menu for navigation
- create MenuLink component for individual styling
- added several mui dependencies 

## Issues closed
close #3

### CODE REVIEW

File: src/organisms/header/index.jsx [hunk 1]
```diff
@@ -6,7 +6,7 @@ const Header = () => {
 		<header>
 			<nav data-test-id="navigation">
 				<Link href="/">Home</Link>
-				<Link href="/about">About me</Link>
+				<Link href="/menu">menu</Link>
 			</nav>
 		</header>
 	);
```
Comment (marcbendixen) [inline src/organisms/header/index.jsx:12]: Are you using this header anywhere 🧐? If not, you could remove it.

File: src/pages/menu/index.jsx [hunk 2]
```diff
@@ -0,0 +1,86 @@
+import * as React from "react";
+import Grid from "@mui/material/Grid";
+import Box from "@mui/material/Box";
+import Paper from "@mui/material/Paper";
+import { experimentalStyled as styled } from "@mui/material/styles";
+import Container from "@mui/material/Container";
+import PeopleAltIcon from "@mui/icons-material/PeopleAlt";
+import { Settings, Logout, Route } from "@mui/icons-material";
+import QrCode2Icon from "@mui/icons-material/QrCode2";
+import CoPresentIcon from "@mui/icons-material/CoPresent";
```
Comment (marcbendixen) [inline src/pages/menu/index.jsx:10]:
```suggestion
import { Settings, Logout, Route, PeopleAltIcon, QrCode2Icon, CoPresentIcon } from "@mui/icons-material";
```

**Review by marcbendixen — APPROVED** [review]
Good job 🚀 

One thing you can also ask Greg: is there an easier way to style in MUI other than a mix of different ways (`sx`, theme, props). (like in your `pages/menu/index.jsx`

File: src/pages/menu/index.jsx (hunk 2 above)
Comment (Marc-Klein) [reply src/pages/menu/index.jsx:10]: Thanks for the info, I'll ask him as soon he's ready. ✌️

File: src/organisms/header/index.jsx (hunk 1 above)
Comment (Marc-Klein) [reply src/organisms/header/index.jsx]: Done ✅

**Pull request merged**